app/models/work.rb

```diff
@@ -0,0 +1,43 @@
+class Work < ApplicationRecord
+  has_many :votes
+  validates :title, presence: true, uniqueness: true
+
+
+  def self.total_votes
+    self.left_outer_joins(:votes).group(:id).order(Arel.sql('COUNT(votes.id) DESC'))
+  end
+
+
+  def self.ranked_media(category)
+    category = category.downcase
+    all_media = self.where(category: category)
+    votes_by_work = all_media.total_votes
+  end
+
+
+  def self.top_ten(category)
+    top_ten = self.ranked_media(category)
+    top_ten.first(10)
+  end
```

Comment:
I love the way the methods in this model build on each other little by little - this is a great example of functional decomposition.